projects/standard-rulesets/src/examples/constants.ts

```diff
@@ -0,0 +1 @@
+export const appliesWhen = ['addedOrChanged', 'always'] as const;
```

Comment:
Is there a ready we wouldn't have added or changed? This is inconsistent with our naming checks ruleset

Reply:
for examples the changes case doesn't really make sense. That's equivalent to always 

same is actually true for naming. Since name is part of the conceptual path there's no way a change rule would ever trigger for changing a name -- ironically. we should just remove it there too